src/components/services/Emby.vue

```diff
@@ -40,15 +40,34 @@ export default {
     seriesCount: 0,
     episodeCount: 0,
   }),
-  computed: {
+  computed: {    
     embyCount: function () {
-      if (this.item.libraryType === "music")
-        return `${this.songCount} songs, ${this.albumCount} albums`;
-      else if (this.item.libraryType === "movies")
-        return `${this.movieCount} movies`;
-      else if (this.item.libraryType === "series")
-        return `${this.episodeCount} eps, ${this.seriesCount} series`;
-      else return `wrong library type 💀`;
+      const libraryType = this.item.libraryType;
```

Comment:
We can avoid a breaking change with somethong like this
```suggestion
      // ensure retro-compatibility with previous implementation 
      const libraryType = Array.isArray(this.item.libraryType) ? this.item.libraryType : [this.item.libraryType];
```